### ISSUE TITLE
Support excluding directories

### ISSUE THREAD

**Issue by radek-holy** [htmllint/htmllint-cli#12]
Hello,

I'd like to run htmllint on my project. However, running `htmllint **/*.html` gives a lot of false positives since it lints also the HTML files of the project dependencies stored in `./node_modules`. The only workaround I can think of is to list all the directories (e.g. `foo/**/*.html`, `bar/**/*.html`) and the top-level HTML files (i.e. `*.html`). But with this approach, there is a big chance that I'll forget to add a newly created directories in the future and I won't notice that they are not linted.

It would be nice if there was a way to list directories/globs that should be excluded (like what you can do with `.gitignore`).

**Comment by radek-holy** [comment]
OK, I don't need this any more (I'm going to use something like `find ... | xargs ...`). But maybe some one else does...

**Comment by kevincolten** [comment]
@radek-holy  What's the glob pattern you used? I have
```sh
./node_modules/.bin/htmllint {*.html,./!(node_modules)/**/*.html}
```
and it kind of works

**Comment by Nxt3** [comment]
You can just do `htmllint $(find /directories /you /want /to /search -name '*.html)`

**Comment by dietergeerts** [comment]
Any plans on including the possibility to use `.gitignore`? `eslint` and `stylelint` have the option

**Comment by iansantagata** [comment]
Late to the party, but for anyone looking for a static glob pattern without having to use `find` and if your `node_modules` are in the root repository, you can do this:

`htmllint {,!(node_modules)/**/}*.html`

Would be nice if this could be supported by a feature like `--ignore-path <path>` similar to ESLint though.
